Access Engineering.js

```diff
@@ -100,9 +100,11 @@ function scrape(doc, url) {
 		let author = ZU.xpath(doc, '//ul[@class="contributor-list"]//li//a');
 		if (author.length > 0) {
 			// Handled using data attribute
+			// Get first name, lastname and author type(Author/Editor etc...)
 			for (let i = 0; i < author.length; i++) {
 				item.creators[i].firstName = author[i].getAttribute('data-firstnames');
 				item.creators[i].lastName = author[i].getAttribute('data-surname');
+				item.creators[i].creatorType = author[i].getAttribute('data-authortype');
 			}
 		}
```

Comment:
@zoe-translates 
Yes correct. I think we don't required any additional code to overwrite creatorType in script file and it will be extract from Embedded Metadata.

Previously we have Markup service supplied author type as author only even it has editor. But now they changed meta author type based on Author type(Editor/Author) from Markup service. Might be I haven't noticed this info.

This PR changes no longer required and I will close the PR. 

Thank you so much for the clear information.

Reply:
@zoe-translates 
Yes its correct.

As per customer requested in Book page(https://www.accessengineeringlibrary.com/content/book/9781259860225) we should show only Editor type and Chapter(https://www.accessengineeringlibrary.com/content/book/9781259860225/toc-chapter/chapter3/section/section1) we have to show both Author and Editor. So we have handled all through Markup service and update in the meta tag.

Reply:
@zoe-translates - We got the issue from meta data overwrite in the code level in the website. We will correct and validate all book and chapter/section in meta tag as you suggested below from Embedded Metadata.

Thank you for the detail information.